draco/synthesis/lyalpha.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+
+from caput import config
+from cora.signal.lssutil import lognormal_transform
+
+from ..core.task import SingleTask
+from ..core.containers import Map
+
+
+def _fgpa(delta, z):
+    """Fluctuating Gunn-Peterson Approximation.
+    See arXiv:1201.0594, arXiv:1509.07875, and https://iopscience.iop.org/article/10.1086/305289/pdf
+    """
+
+    A = 0.3 * ((1 + z) / 3.4) ** 4.5  # arXiv:1509.07875 sec. 3
+    alpha = 1.6
+    # Fbar = 0.64  # empirical mean transmission from Croft et al.
+
+    # take lognormal transform to avoid negative optical depths
+    delta = lognormal_transform(delta, axis=1)
+
+    F = np.exp(-A[:, np.newaxis] * (1 + delta) ** alpha)
+    F /= F.mean(axis=1)[:, np.newaxis]
+
+    return F
+
+
+class GenerateFluxTransmission(SingleTask):
+    # TODO: should this use the HI density or matter density as an input?
```

Comment:
Good question. I guess really it's the HI density, though you'd have the most flexibility if you just started from the matter density. 

Also this needs docstrings.